### DISCUSSION TITLE
Mux-Player-React vs Mux-Video-React

### DISCUSSION THREAD

**Discussion by jazsouf** [muxinc/elements#983] [Q&A] [answered]
Hey,

I was looking into how to implement a performant video player in Next.js. I was reading through the code from the next-video library and ended up here. 

I am having a hard time assessing the differences between mux-video-react and mux-player-react.

mux-video-react returns the HTML <video> tag while the mux-player-react returns <mux-player> component.

Do the same packages still load HLS and have performance built-in? Does the mux-player-react just add a layer of styling and theming?

I will test both to see what's up but if anyone here has knowledge about this, any detail would be great to know!

**Comment by luwes** [comment, marked as answer]
> Do the same packages still load HLS and have performance built-in? Does the mux-player-player just add a layer of styling and theming?

Indeed, this is the case. There are some other small details but generally yes mux-player-react adds the Mux themed UI on top of mux-video-react. 

standalone mux-video-react is capable of loading HLS sources via the hls.js engine and also has the Mux data tracking built in.
